packages/curl.rb

```diff
@@ -212,7 +212,9 @@ def self.install
     # Curl already includes man pages via "curl -M"
     FileUtils.rm_rf "#{CREW_DEST_PREFIX}/musl/share/man"
     FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
-    FileUtils.ln_s "#{CREW_DEST_PREFIX}/musl/bin/curl", "#{CREW_DEST_PREFIX}/bin/curl"
+    Dir.chdir "#{CREW_DEST_PREFIX}/bin" do
```

Comment:
Change to
```
FileUtils.ln_s '../musl/bin/curl' "#{CREW_DEST_PREFIX}/bin/curl"
```

Reply:
`FileUtils.ln_s "#{CREW_PREFIX}/musl/bin/curl", "#{CREW_DEST_PREFIX}/bin/curl"` also works and clarifies the location more clearly.  Relative symlinks are not important because a relocated `#{CREW_PREFIX}` will need to be recompiled anyway.

Reply:
Guys, for whatever reason, this kept breaking the symlinks. So I left this code in as I don't have time to figure out why crew is breaking this.
<img width="662" alt="Screen Shot 2021-11-04 at 12 20 29 PM" src="https://user-images.githubusercontent.com/1096701/140383232-f904a504-533c-450d-a6a5-3f8239ba2e93.png">
.